safe/gis/vector/multi_buffering.py

```diff
@@ -122,8 +123,9 @@ def multi_buffering(layer, radii, callback=None):
             callback(current=i, maximum=feature_count, step=processing_step)
 
     # We transfer keywords to the output.
-    buffered.keywords = layer.keywords
+    buffered.keywords = KeywordIO().read_keywords(layer)
```

Comment:
I thought you want the keywords wizard will automatically give the default option on the dialog, so I think I could get it from the source's keywords. Or maybe it's just my misunderstanding :joy: so, stick with the old one?